api/src/main/java/org/openmrs/api/db/PatientDAO.java

```diff
@@ -260,13 +260,27 @@
 	public Allergies saveAllergies(Patient patient, Allergies allergies);
 	
 	/**
-	 * Gets a allergy
+	 * Gets a allergy matching the given allergyId
```

Comment:
I think it should be "the allergy" instead of "a allergy". Not so? @jdegraft  :)